charts/kuzzle/Chart.yaml

```diff
@@ -14,7 +14,7 @@ type: application
 
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
-version: 1.1.0
+version: 1.1.1
```

Comment:
This should be version 1.2.0 since this is a new feature

Reply:
If we're going to bump the minor version number then let's also update the version of Kuzzle that is deployed by this chart?

Reply:
Yea, you are right, you mean appVersion: 2.25.0 -> appVersion: 2.28.0 right ?

Reply:
Yeah actually we should use the `appVersion` as the tag of the image, so the image that is pulled is the one the user expects by looking at this value.

Reply:
Well good news

* https://github.com/kuzzleio/kuzzle/blob/master/.github/workflows/kuzzle-core.yml
* https://hub.docker.com/layers/kuzzleio/kuzzle/2.28.0/images/sha256-949e7a2bcf37aeb30302e42cec50356a9c2f52be2a95bb3259809ebce4c29cf1?context=repo

Workflow already do that 👍🏼 we can do it now, is it okay for you to update it ? :)